[2025-04-17T23:28:23.251+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: parquet_loader.create_tables manual__2025-04-17T23:28:21.296730+00:00 [queued]>
[2025-04-17T23:28:23.257+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: parquet_loader.create_tables manual__2025-04-17T23:28:21.296730+00:00 [queued]>
[2025-04-17T23:28:23.257+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-04-17T23:28:23.266+0000] {taskinstance.py:1382} INFO - Executing <Task(SQLExecuteQueryOperator): create_tables> on 2025-04-17 23:28:21.296730+00:00
[2025-04-17T23:28:23.269+0000] {standard_task_runner.py:57} INFO - Started process 277 to run task
[2025-04-17T23:28:23.271+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'parquet_loader', 'create_tables', 'manual__2025-04-17T23:28:21.296730+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/parquet_to_postgres_dbt.py', '--cfg-path', '/tmp/tmp8wkgk2vt']
[2025-04-17T23:28:23.273+0000] {standard_task_runner.py:85} INFO - Job 22: Subtask create_tables
[2025-04-17T23:28:23.319+0000] {task_command.py:416} INFO - Running <TaskInstance: parquet_loader.create_tables manual__2025-04-17T23:28:21.296730+00:00 [running]> on host fe2f3e6bd690
[2025-04-17T23:28:23.391+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='parquet_loader' AIRFLOW_CTX_TASK_ID='create_tables' AIRFLOW_CTX_EXECUTION_DATE='2025-04-17T23:28:21.296730+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-17T23:28:21.296730+00:00'
[2025-04-17T23:28:23.392+0000] {sql.py:274} INFO - Executing: 
        CREATE TABLE IF NOT EXISTS staging_table (
            id INT,
            name TEXT,
            age INT
        );
        
        CREATE TABLE IF NOT EXISTS main_table (
            id INT PRIMARY KEY,
            name TEXT,
            age INT
        );
        
[2025-04-17T23:28:23.401+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2025-04-17T23:28:23.411+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2025-04-17T23:28:23.415+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/operators/sql.py", line 280, in execute
    output = hook.run(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/hooks/sql.py", line 385, in run
    with closing(self.get_conn()) as conn:
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/postgres/hooks/postgres.py", line 155, in get_conn
    self.conn = psycopg2.connect(**conn_args)
  File "/home/airflow/.local/lib/python3.8/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "postgres" (172.18.0.2), port 5432 failed: FATAL:  password authentication failed for user "postgres"

[2025-04-17T23:28:23.426+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=parquet_loader, task_id=create_tables, execution_date=20250417T232821, start_date=20250417T232823, end_date=20250417T232823
[2025-04-17T23:28:23.434+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 22 for task create_tables (connection to server at "postgres" (172.18.0.2), port 5432 failed: FATAL:  password authentication failed for user "postgres"
; 277)
[2025-04-17T23:28:23.452+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-04-17T23:28:23.475+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
